site/mkdocs.yml

```diff
@@ -20,14 +21,14 @@ theme:
 extra_css:
   - stylesheets/extra.css
 
-google_analytics:
-  - UA-177850801-1
-  - auto
 extra:
   versions:
     java: 0.6.1
     python: v0.6.1
     iceberg: 0.11.1
+  analytics:
```

Comment:
note: this change hasn't been released in mkdocs-material so we are likely to lose analytics for a day or two. The mkdocs-material release will happen any day now.